crowdin_content.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# Your crowdin's credentials
+#
+"project_identifier": "cs-unplugged"
+"api_key_env": CROWDIN_API_KEY
+"base_path": ""
+#"base_url" : ""
+
+#
+# Choose file structure in crowdin
+# e.g. true or false
+#
+"preserve_hierarchy": true
+
+#
+# Files configuration
+#
+files: [
+    {
+        "source": "/csunplugged/topics/content/en/**/*.yaml",
+        "translation": "/csunplugged/topics/content/%two_letters_code%/**/%original_file_name%",
+
+        # The parameter "update_option" is optional. If it is not set, translations for changed strings will be lost. Useful for typo fixes and minor changes in source strings.
+        # e.g. "update_as_unapproved" or "update_without_changes"
+        #
+        #"update_option" : "",
```

Comment:
This was part of the generated skeleton - I decided to leave it here because this is an option that we are likely to use, and it's useful having a reminder that it is available. 

Happy to remove if you disagree.